### DISCUSSION TITLE
adding Music (Mac)

### DISCUSSION THREAD

**Discussion by justkatia** [onlaj/Piano-LED-Visualizer#236] [Q&A]
How can I add Music to the PI with my Mac? I don't know which programs I could use and how and where to upload the midi files. btw thanks for sharing the instructions <3

**Comment by justkatia** [comment]
Update: I use FileZilla as a program but when I wan't to upload a midi file there is an error: "open for write: permission denied
Error: File transfer failed"

**Reply by onlaj** [reply]
Try to change permission, connect to RPi through SSH and type

`sudo chown pi:pi -R /home/Piano-LED-Visualizer/Songs`
